apps/web/playwright/insights.e2e.ts

```diff
@@ -0,0 +1,98 @@
+import { expect } from "@playwright/test";
+
+import { randomString } from "@calcom/lib/random";
+import prisma from "@calcom/prisma";
+
+import { test } from "./lib/fixtures";
+
+test.describe.configure({ mode: "parallel" });
+
+const createTeamsAndMembership = async (userIdOne: number, userIdTwo: number) => {
```

Comment:
There is another PR that have more test here, reused some logic but only added required test here.